Assets/Mirror/Runtime/NetworkWriterPool.cs

```diff
@@ -21,6 +21,8 @@ public void Dispose()
     /// </summary>
     public static class NetworkWriterPool
     {
+        static readonly ILogger logger = LogFactory.GetLogger(typeof(NetworkWriterPool));
```

Comment:
Same as NetworkReaderPool, set the default log level to error, since previously the log was only enabled when LogFilter.Debug is true